simple/tests/stats/events_importer_test.py

```diff
@@ -38,14 +39,6 @@
 _EXPECTED_DIR = os.path.join(_TEST_DATA_DIR, "expected")
 
 
-def _compare_files(test: unittest.TestCase, output_path, expected_path):
-  with open(output_path) as gotf:
-    got = gotf.read()
-    with open(expected_path) as wantf:
-      want = wantf.read()
-      test.assertEqual(got, want)
-
-
 def _write_observations(db_path: str, output_path: str):
```

Comment:
TODO: could _write_triples also be in the helper (I see it dup'd)?

Reply:
Ack. Will do.